src/syft/lib/zksk/secret.py

```diff
@@ -0,0 +1,38 @@
+# stdlib
+
+# third party
+import petlib  # noqa: 401
+from petlib.bn import Bn
+import zksk
+
+# syft relative
+from ...generate_wrapper import GenerateWrapper
+from ...proto.lib.petlib.ec_obj_pb2 import EC2_obj
```

Comment:
This is a different library so I would just change this entirely to not use the same protobuf.
We already have a generic protobuf for when you want to just send bytes, which `tenseal` is also using.

```proto
message VendorBytes {
  string obj_type = 1;
  string vendor_lib = 2;
  string vendor_lib_version = 3;
  bytes content = 4;
}
```